src/D2L.CodeStyle.Analysis/MutabilityInspector.cs

```diff
@@ -125,7 +125,7 @@ HashSet<ITypeSymbol> typeStack
 
 				{
 					foreach( ISymbol member in type.GetMembers() ) {
-						if( IsMemberMutableRecursive( member, typeStack ) ) {
+						if( !member.IsStatic && IsMemberMutableRecursive( member, typeStack ) ) {

```

Comment:
I do not follow how you can say a type with only static members is immutable. Can you explain?